helpers/index.d.ts

```diff
@@ -0,0 +1 @@
+import "./multi";
```

Comment:
Would it maybe be simpler to just put all helpers into one file (and maybe rename it to `helpers.d.ts`), and check it in code based on declaration name?

Reply:
I've merged helpers into `helpers/index.d.ts` for now since I can't quite figure out how to adjust the test builder factory to just use `helpers.d.ts` it seems to insist on using `helpers/index.d.ts` and I've yet to find a workaround so far